src/constants.js

```diff
@@ -23,4 +23,5 @@ export default {
         ANALYSIS: "Analysis",
     },
     IPLANT: "iplantcollaborative",
+    WEBSOCKET_MAX_CONNECTION_ATTEMPTS: 10,
```

Comment:
I also agree this might be better suited in the config file

Reply:
Hmm I am not sure if it worth the effort  to put this in a config file since this is needed on the client side. Since this is not in a page, there isn't a straightforward way to read the configs from server side as far as I know.  I don't remember changing this value in DE ever since it we switched to websockets. I am gonna leave it here until we find a good reason to move this to configs.